src/hooks/connectWeb3/useWeb3React.tsx

```diff
@@ -74,14 +79,21 @@ export function ActiveWeb3Provider({
   jsonRpcEndpoint,
   children,
 }: PropsWithChildren<ActiveWeb3ProviderProps>) {
-  const integratorConnection = useMemo(() => getWallet(propsProvider), [propsProvider])
+  const [error, setError] = useState<Error>()
+
+  useEffect(() => error && console.warn(error), [error])
```

Comment:
Will this just _log_ the error?

Reply:
If this is just meant to log the error, can you avoid adding more state, as it will slow down the application.

Instead, consider a callback - in this case it can be a free function, or you can just pass in `console.warn`.